android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -748,6 +774,11 @@ public void downloadFromAssets() {
             //TODO: Add and Take counter value from to config.json
             sharedPreferences.edit().putBoolean(ConstantStrings.DATABASE_RECORDS_EXIST, true).apply();
 
+            startDownloadListener();
+            Timber.d("JSON parsing started");
+
+            OpenEventApp.postEventOnUIThread(new CounterEvent(6)); // Bump if increased
```

Comment:
We neither load it from the internet, nor from JSON

Please create an issue for this. To be solved later